slack_and_notifications.md

```diff
@@ -20,25 +20,14 @@ it for you communicating the `[project-name]`.
 
 :warning: **You must have already setup [Automatic Deployment through SemaphoreCI](ruby_on_rails/configure_cd.md)** :warning:
 
-On Slack:
-1. Create a new Custom Integration --> Incoming Webhook
-2. Post to the channel `#project-[project-name]`
-3. Copy the Webhook URL and note it
-4. In "Customize Name" write "Semaphore CI"
-5. In "Customize Icon" place this icon: ![semaphore_icon](images/semaphore_icon.png)
-6. Save
-
-Now proceed to SemaphoreCI:
-
-1. Open the project Slack notifications settings (`https://semaphoreci.com/renuo/[project-name]/settings/notifications/slack/new`)
-2. Paste the Webhook URL
-3. Change **Receive after builds** to **Never**
-4. Select only the `[project-name]-master` server
-5. Include the Project Name
-6. Save
+1. Open the project Notifications settings (`https://renuo.semaphoreci.com/notifications`)
```

Comment:
updated for Semaphore 2

Reply:
isn;t this done automatically from the renuo-cli configure-semaphore command? I think so...

Reply:
yes, it is. I added a sentence saying that you need to do this only if you configured semaphore manually, without cli.